pdbfixer/pdbfixer.py

```diff
@@ -953,21 +953,28 @@ def addMissingAtoms(self, seed=None):
             context.setPositions(newPositions)
             mm.LocalEnergyMinimizer.minimize(context)
             state = context.getState(getPositions=True)
```

Comment:
Everything from here up to the end of the `if newTopology.getNumResidues() > 1:` block is a pretty sensible self-contained chunk of behaviour. It might make sense to pull this out into a new method? So you would just have something like `state = self._pushOverlyCloseAtomsApart(newTopology, newAtoms, integrator, context)`. Probably with a better name :)